samples/DispatcherSample.Web/CorsApis.cs

```diff
@@ -0,0 +1,666 @@
+﻿using System;
```

Comment:
You can ignore this class as its a copy of some apis from Cors. You can take a look at file `StartupUsingMyCorsMiddleware` to see the usage. Its pretty straightforward

Reply:
Add a comment for future readers about where this is copied from.
Also add the standard ASP.NET copyright header to all the cs files.